core/tests/protractor_utils/workflow.js

```diff
@@ -121,6 +121,8 @@ var createAndPublishExploration = function(
 
 // Here, 'roleName' is the user-visible form of the role name (e.g. 'Manager').
 var _addExplorationRole = function(roleName, username) {
+  element(
+    by.model('explorationTitleService.displayed')).sendKeys('Chuck Norris');
```

Comment:
Just a suggestion, you can break at the next bracket I think.
 
element(by.model('explorationTitleService.displayed')).sendKeys(
   'Chuck Norris');

Reply:
Alright, I agree, I will do it.
Thanks!